### PR TITLE
Fix typos and improve clarity in FAQ

### PR DESCRIPTION
**Pull request by embh**
This PR includes typo fixes and some proposed changes to sections that I found unclear. I've added a few questions below.

### CODE REVIEW

File: dpc-web/app/views/pages/faq.html.erb [hunk 1]
```diff
@@ -194,12 +194,12 @@ Already using or testing wth FHIR, especially Bulk FHIR</li>
       </button>
     </h2>
     <div id="a13" class="usa-accordion__content">
-      <p>Yes! The emergency room scenario is a great use of claims data. When adding patients to the roster from an emergency room, the time period will be 72 hours instead of the 90 days that occurs for outpatient visits. Otherwise, you can add patients to your roster as they are admitted to the emergency room.</p>
+      <p>Yes! The emergency room scenario is a great use of claims data. When adding patients to the roster from an emergency room, the attribution time period will be 72 hours instead of the 90 days that occurs for outpatient visits.</p>
```
Comment (embh) [inline dpc-web/app/views/pages/faq.html.erb]: > 72 hours
The terms of service say two days. Which is correct?

**Comment by embh** [comment]
Can we make this clearer?
> CMS will be using the pilot to test notification and consent messages and processes that are easy to understand by beneficiaries without adding additional complexity to the sharing of data.

https://github.com/CMSgov/dpc-app/pull/520/files#diff-b961714e94ad2e5d931f18567c50d5a2L127

File: dpc-web/app/views/pages/faq.html.erb (hunk 1 above)
Comment (switzersc-usds) [reply dpc-web/app/views/pages/faq.html.erb]: cc @amgleason

**Comment by switzersc-usds** [comment]
> Can we make this clearer?

Agreed. I'm not sure what exactly it's saying either. cc @amgleason

File: dpc-web/app/views/pages/faq.html.erb (hunk 1 above)
Comment (embh) [reply dpc-web/app/views/pages/faq.html.erb]: @amgleason recommended removing. I've changed it to say only that patients can be added to a roster from the ER, with no time period.

**Pull request merged**